### PR TITLE
Reader Refresh: Set line-clamp with pixels instead of -webkit-line-clamp

### PR DESCRIPTION
**Pull request by samouri**
Firefox + IE don't understand -webkit-line-clamp so this PR aims to solve a bug where Search recs overflow far too far.

Before in FF:
FF broken

After in FF:
FF fixed

Solves the reason we care about: #9672

### CODE REVIEW

File: client/reader/search-stream/style.scss [hunk 1]
```diff
@@ -34,6 +34,11 @@
 	width: 830px;
 }
 
+/* clamp to 3 lines for search-rec excerpts */
+ .is-reader-page .search-stream .reader-related-card-v2__excerpt.post-excerpt {
+	max-height: 15px * 1.6 * 3;
```
Comment (blowery) [inline client/reader/search-stream/style.scss]: should we also include the `long-content-fade` here, like we do for the related cards in the main stream?

Comment (samouri) [inline client/reader/search-stream/style.scss]: I believe `long-content-fade` is already being applied.

Comment (bluefuton) [inline client/reader/search-stream/style.scss]: Looks like it's being applied, but perhaps starts too late for these cards? It often seems to kick in after the `...`:

screen shot 2016-12-12 at 14 36 35

**Comment by bluefuton** [comment]
Tiny nit: descenders (e.g. on lowercase _g_ and _j_) are being cropped in FF, so we might need to tweak the height very slightly:

screen shot 2016-12-12 at 14 35 53

**Comment by bluefuton** [comment]
Since we do this in several places, a `@line-clamp` SASS mixin might be a good idea. Happy if we address that in a separate PR though.

**Comment by samouri** [comment]
Closing because solved in #9962

**Pull request closed without merging**